pegasus/sites.v3/code.org/views/video_player.js.haml

```diff
@@ -1,13 +1,12 @@
-%script{src: "/js/videojs-ie8.min.js"}
-%script{src: "/js/video.min.js"}
-
 %link{href: "/css/video-js.css", rel: "stylesheet"}
 
 :javascript
 
   $(window).load(function() {
     if (window.location.search.indexOf("force_youtube_fallback") !== -1)
     {
+      $.getScript("/js/videojs-ie8.min.js");
+      $.getScript("/js/video.min.js");
```

Comment:
1. This line of code looks like it will only run when the explicit query parameter (`force_youtube_fallback`) is provided, it won't be run in the normal case when `setupVideos("fallback")` is executed as a result of `testImageAccess` executing the `failureCallback`. This will cause the video player to fail in the normal case.

    I think this code need to be moved elsewhere in order for the logic to work correctly in all cases (maybe [line `37`/`38` below](https://github.com/code-dot-org/code-dot-org/pull/15156/files#diff-5c242054d57287c8dc18a6d41b65ac71R37)?)